CHANGELOG.md

```diff
@@ -1,99 +1,98 @@
-# Revision History
-
-## 1.6 (unreleased)
+# 1.6 (unreleased)
 
 - Added `git svn` support. (@daniel-brosche)
```

Comment:
In my opinion worth to mention the following:

- Added `--skip-changes` option to skip changed dependencies on `ìnstall` or `update`

Reply:
Definitely! Feel free to send a quick PR if I don't get to that before the next release.